pkg/cri/server/container_create_linux.go

```diff
@@ -260,7 +261,12 @@ func (c *criService) containerSpec(
 	// Get RDT class
 	rdtClass, err := rdt.ContainerClassFromAnnotations(config.GetMetadata().GetName(), config.Annotations, sandboxConfig.Annotations)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to set RDT class")
+		if !rdt.Enabled() && c.config.ContainerdConfig.IgnoreRdtNotEnabledErrors {
+			rdtClass = ""
+			logrus.Info(err)
```

Comment:
how about logrus.Debugf("continuing create container %s, ignoring rdt not enabled %v", containerName, err)

Reply:
changed to
```
logrus.Debugf("continuing create container %s, ignoring rdt not enabled (%v)", containerName, err)
```